core/src/main/resources/defaults/clustertemplates/7.2.10/aws/edgenode.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "7.2.10 COD Edge Node for AWS",
+  "description": "",
+  "type": "OPERATIONALDATABASE",
+  "featureState": "RELEASED",
```

Comment:
I'm unfamiliar with the `type`'s values in the cluster templates. Does it have any _real_ effect on the created DataHub? These edge node cluster tempaltess are related to the OPERATIONALDATABASE area but are not directly OPERATIONALDATABASE DataHubs.

The "featureState": "RELEASED" is also questionable in this context but as I know the other value is IN_PREVIEW so going with RELEASED is better.

Reply:
As far as I know OPERATIONALDATABASE is just an enum that we use on the UI, it gives more context to the user.